modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2163,6 +2164,12 @@
 		end
 	end
 
+	function m.supportOpenMP(cfg)
+		if cfg.openmp == "On" then
```

Comment:
I think it is unnecessary to check Visual Studio version here.

Reply:
I know there's OpenMP support back at least as far as 2015, so I'd be happy with no version check.

Reply:
I learned that VS2010 started to support OpenMP and premake only supports VS2013 and above versions. So it's OK not to check it.